packages/tokens/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+    "extends": "@workleap/typescript-configs/library.json",
+    "include": ["style-dictionary/build.ts"],
+    "exclude": ["node_modules", "build"],
+    "compilerOptions": {
+        "outDir": "build",
```

Comment:
we usually use tsup to build instead of typescript. 
Thats why the config has noEmit true by default 

simplest example should be here: https://github.com/workleap/wl-web-configs/tree/main/packages/stylelint-config.